TreeTracker/UI/Navigation/Home/HomeCoordinator.swift

```diff
@@ -111,6 +127,10 @@ extension HomeCoordinator: HomeViewModelCoordinatorDelegate {
     func homeViewModel(_ homeViewModel: HomeViewModel, didSelectUploadListForPlanter planter: Planter) {
         showUploadList(planter: planter)
     }
+
+    func homeViewModel(_ homeViewModel: HomeViewModel, didLogoutPlanter planter: Planter) {
+        logoutPlanter(planter: planter)
```

Comment:
The HomeCoordinatorDelegate has a function called `homeCoordinatorDidLogout(_ homeCoordinator: HomeCoordinator)`, you should just need to call that here.

The RootCoordinator responds to this and restarts the AuthenticationCoordinator when it is called.